pages/stats/StatsHandler.php

```diff
@@ -27,9 +33,109 @@ class StatsHandler extends PKPStatsHandler
     public function __construct()
     {
         parent::__construct();
+        $this->addRoleAssignment(
+            [Role::ROLE_ID_SITE_ADMIN, Role::ROLE_ID_MANAGER, Role::ROLE_ID_SUB_EDITOR],
+            ['issues']
+        );
         Hook::add('TemplateManager::display', [$this, 'addSectionFilters']);
     }
 
+    /**
+     * Display issues stats
+     *
+     */
+    public function issues(array $args, Request $request): void
+    {
+        $dispatcher = $request->getDispatcher();
+        $context = $request->getContext();
+
+        if (!$context) {
+            $dispatcher->handle404();
+        }
+
+        $templateMgr = TemplateManager::getManager($request);
+        $this->setupTemplate($request);
+
+        $dateStart = date('Y-m-d', strtotime('-31 days'));
+        $dateEnd = date('Y-m-d', strtotime('yesterday'));
+        $count = 30;
+
+        $timeline = Services::get('issueStats')->getTimeline(PKPStatisticsHelper::STATISTICS_DIMENSION_DAY, [
+            'assocTypes' => [Application::ASSOC_TYPE_ISSUE],
+            'contextIds' => [$context->getId()],
+            'count' => $count,
+            'dateStart' => $dateStart,
+            'dateEnd' => $dateEnd,
+        ]);
+
+        $statsComponent = new \APP\components\StatsIssuePage(
+            $dispatcher->url($request, PKPApplication::ROUTE_API, $context->getPath(), 'stats/issues'),
+            [
+                'timeline' => $timeline,
+                'timelineInterval' => PKPStatisticsHelper::STATISTICS_DIMENSION_DAY,
+                'timelineType' => 'toc',
+                'tableColumns' => [
+                    [
+                        'name' => 'title',
+                        'label' => __('editor.issues.issueIdentification'),
+                    ],
+                    [
+                        'name' => 'tocViews',
+                        'label' => __('stats.issueTOCViews'),
```

Comment:
> Views

Reply:
Shell I change the locale string, so it is "Views" also in the CSV file? Or just here?

Reply:
Yeah, let's use "views" and "downloads" everywhere for issues.